server/utils/applications/utils.ts

```diff
@@ -56,18 +57,57 @@ export {
 } from './pendingPlacementRequestTable'
 export { isWomensApplication } from './isWomensApplication'
 
-const applicationTableRows = (applications: Array<ApplicationSummary>): Array<TableRow> => {
+const applicationTableRows = (applications: Array<Cas1ApplicationSummary>): Array<TableRow> => {
   return applications.map(application => [
-    createNameAnchorElement(application.person, application),
+    {
+      ...createNameAnchorElement(application.person, application),
+      attributes: { 'data-sort-value': displayName(application.person) },
+    },
     textValue(application.person.crn),
     htmlValue(getTierOrBlank(application.risks?.tier?.value?.level)),
```

Comment:
I think this will need a data-sort-value added although I don't know what that should be. I believe that the sort order for tier has the alpha part running in the opposite direction to the numeric part, so to sort correctly, I _think_ it would need to be completely transformed.  Maybe map A..Z -> Z..A for instance.